### DISCUSSION TITLE
Ошибка при создании симлинков

### DISCUSSION THREAD

**Discussion by kanishev** [doka-guide/platform#725] [Q&A] [answered]
Написал статью. Теперь хочу посмотреть, как она будет выглядеть локально у себя.
Склонировал platform директорию, установил зависимости. При попытке установки симлинков выскакивает ошибка. 

![image](https://user-images.githubusercontent.com/50417170/140103332-01023d38-380e-48cd-933d-de2c2221654d.png)

Подскажите, пожалуйста, в чем может быть проблема. Перелопатил все что можно, все печально. Делал все по видео Вадима.

**Comment by furtivite** [comment, marked as answer]
Привет, уточни, пожалуйста, на машине установлен Windows? Если да, то терминал должен быть запущен от имени администратора.

**Reply by kanishev** [reply]
Спасибо. Это помогло!
